### PR TITLE
Change RTD to use pip instead of conda

### PR DESCRIPTION
**Pull request by larrybradley**

### CODE REVIEW

File: readthedocs.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
-conda:
-  file: .rtd-environment.yml
-
 python:
-   setup_py_install: true
+  version: 3.5
```
Comment (pllim) [inline readthedocs.yml]:
```suggestion
  version: 3.7
```
Python 3.7 is faster, if available.

Comment (pllim) [reply readthedocs.yml]: Actually, nvm. I vaguely remember some Sphinx stuff not working in 3.7. Try 3.6.

**Comment by larrybradley** [comment]
Indeed, but it *should* work!

**Comment by pllim** [comment]
🤞

**Comment by larrybradley** [comment]
RTD failed.  I just updated the docs config.

**Pull request merged**